src/osd/OSDMap.cc

```diff
@@ -3968,7 +3968,13 @@ void OSDMap::print_summary(Formatter *f, ostream& out,
     f->dump_int("epoch", get_epoch());
     f->dump_int("num_osds", get_num_osds());
     f->dump_int("num_up_osds", get_num_up_osds());
+    if (last_up_change != utime_t()) {
+      f->dump_string("osd up since", utimespan_str(now - last_up_change));
```

Comment:
Please replace spaces with underscores in field names.
Use dump_int + a number.